src/test/groovy/com/getkeepsafe/dexcount/DexFileTest.groovy

```diff
@@ -0,0 +1,22 @@
+package com.getkeepsafe.dexcount;
+
+import spock.lang.Specification
+
+class DexFileTest extends Specification {
+    def "test AAR dexcount"() {
+        setup:
+        def currentDir = new File(".").getAbsolutePath()
+
+        when:
+        def aarFile = new File(currentDir + File.separatorChar + "src"
+                + File.separatorChar + "test" + File.separatorChar + "resources"
+                + File.separatorChar + "android-beacon-library-2.7.aar")
```

Comment:
Ah, you beat me to my edit.  I missed that you checked in the file, in test resources to boot!

The problem is the way the file is accessed at run time; it needs to be extracted from the jar, e.g. `getClass().getResourceAsStream('android-beacon-library-2.7.aar')`.